[2025-09-14T16:53:17.189+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-14T16:53:17.281+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.analyze scheduled__2025-09-13T16:00:00+00:00 [queued]>
[2025-09-14T16:53:17.304+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.analyze scheduled__2025-09-13T16:00:00+00:00 [queued]>
[2025-09-14T16:53:17.305+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-14T16:53:17.337+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): analyze> on 2025-09-13 16:00:00+00:00
[2025-09-14T16:53:17.358+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'analyze', 'scheduled__2025-09-13T16:00:00+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmp4nc_70if']
[2025-09-14T16:53:17.368+0000] {standard_task_runner.py:105} INFO - Job 271: Subtask analyze
[2025-09-14T16:53:17.367+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=222) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-14T16:53:17.369+0000] {standard_task_runner.py:72} INFO - Started process 223 to run task
[2025-09-14T16:53:17.691+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.analyze scheduled__2025-09-13T16:00:00+00:00 [running]> on host 02c5a69eec0c
[2025-09-14T16:53:18.121+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='analyze' AIRFLOW_CTX_EXECUTION_DATE='2025-09-13T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-13T16:00:00+00:00'
[2025-09-14T16:53:18.123+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-14T16:53:18.153+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-14T16:53:18.156+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/src/analyze.py']
[2025-09-14T16:53:18.181+0000] {subprocess.py:86} INFO - Output:
[2025-09-14T16:53:20.748+0000] {subprocess.py:93} INFO - /opt/***/src/analyze.py:162: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
[2025-09-14T16:53:20.755+0000] {subprocess.py:93} INFO -   today = datetime.utcnow().strftime("%Y-%m-%d")
[2025-09-14T16:53:20.804+0000] {subprocess.py:93} INFO - + Text report saved to /opt/***/data/reports/report_2025-09-14.txt
[2025-09-14T16:53:20.989+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-14T16:53:21.200+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-14T16:53:21.202+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=analyze, run_id=scheduled__2025-09-13T16:00:00+00:00, execution_date=20250913T160000, start_date=20250914T165317, end_date=20250914T165321
[2025-09-14T16:53:21.295+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-14T16:53:21.421+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-14T16:53:21.426+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
